js/main.js

```diff
@@ -95,10 +98,10 @@ async function closeSkill() {
     document.getElementById("skill-title").textContent = "";
     document.getElementById("skill-desc").innerHTML = "";
 }
-  
+
 // Randomly load an image to be used for the 404 meme
 function load404Image() {
-    images = [
+    var images = [
```

Comment:
This stops an error message and makes images a local variable instead of a global variable